confiture-web-app/src/pages/help/HelpPage.vue

```diff
@@ -136,39 +136,32 @@ const moreLinks = [
         >
           <div class="fr-tile fr-enlarge-link fr-tile--horizontal">
             <div class="fr-tile__body">
-              <h3 class="fr-tile__title">
+              <h3>
                 <RouterLink
                   class="fr-tile__link"
                   :to="{ name: link.routeName }"
                   >{{ link.title }}</RouterLink
                 >
               </h3>
-              <p class="fr-tile__desc">{{ link.description }}</p>
+              <p v-if="link.description" class="fr-tile__desc">{{ link.description }}</p>
             </div>
           </div>
         </div>
       </div>
 
-      <h3>Aller plus loin</h3>
-      <p>Retrouver plus d’information les sites officiels :</p>
+      <h2>Aller plus loin</h2>
+      <p>Retrouver plus d’informations sur les sites officiels :</p>
 
       <ul class="fr-mb-7w">
         <li v-for="link in moreLinks" :key="link.title">
-          <a class="fr-link" :href="link.href" target="_blank">{{
+          <a class="fr-link" :href="link.href" :title="
+            link.title + ' - nouvelle fenêtre'" target="_blank">{{
             link.title
           }}</a>
```

Comment:
On peut pas sauter de ligne dans un attribut

```suggestion
          <a
            class="fr-link"
            :href="link.href"
            :title="link.title + ' - nouvelle fenêtre'"
            target="_blank"
          >
            {{ link.title }}
          </a>
```